src/pages/WorkProgress.tsx

```diff
@@ -200,23 +200,25 @@ const WorkProgress = ({ workTypes }: { workTypes: string[] }) => {
                       <DownloadIcon name="Download" className="h-4 w-4 text-sdb" />
                     </a>
                   </div>
-                  <DataTable
-                    sortable
-                    defaultSort={[
-                      //Sort by Status and within status sort with WorkNumber in descending order
-                      {
-                        id: 'status',
-                        desc: false
-                      },
-                      {
-                        id: 'workNumber',
-                        desc: true
-                      }
-                    ]}
-                    columns={columns}
-                    data={searchResult.entries}
-                    ref={sortedTableDataRef}
-                  />
+                  <div className="max-h-[750px] overflow-y-auto overflow-x-auto">
```

Comment:
Is this 750 px  work for all screen sizes ? or do we need something like max-h-screen ?

Reply:
yeah max-h-screen is probably better than an arbitrary number